frontend/src/components/common/form/ContactInput.tsx

```diff
@@ -0,0 +1,96 @@
+import classNames from 'classnames';
+import { Button, Input } from 'components/common/form';
+import { getIn, useFormikContext } from 'formik';
+import { IContactSearchResult } from 'interfaces';
+import React from 'react';
+import { Col, Row } from 'react-bootstrap';
+import Form from 'react-bootstrap/Form';
+import { FormControlProps } from 'react-bootstrap/FormControl';
+import { FaAddressBook } from 'react-icons/fa';
+import { MdClose } from 'react-icons/md';
+import styled from 'styled-components';
+
+import TooltipWrapper from '../TooltipWrapper';
+import { DisplayError } from './DisplayError';
+
+type RequiredAttributes = {
+  field: string;
+  setShowContactManager: React.Dispatch<React.SetStateAction<boolean>>;
+  onClear: Function;
+};
+
+type OptionalAttributes = {
+  label?: string;
+  required?: boolean;
+  displayErrorTooltips?: boolean;
+};
+
+export type ContactInputProps = FormControlProps & OptionalAttributes & RequiredAttributes;
+
+export const ContactInput: React.FC<ContactInputProps> = ({
+  field,
+  setShowContactManager,
+  onClear,
+  label,
+  required,
+  displayErrorTooltips,
+}) => {
+  const { errors, touched, values } = useFormikContext<any>();
+  const error = getIn(errors, field);
+  const touch = getIn(touched, field);
+  const value: IContactSearchResult | undefined = getIn(values, field);
+  const errorTooltip = error && touch && displayErrorTooltips ? error : undefined;
+
+  return (
+    <Form.Group
+      controlId={`input-${field}`}
+      className={classNames(!!required ? 'required' : '', 'input')}
+    >
+      {!!label && <Form.Label>{label}</Form.Label>}
+
+      <TooltipWrapper toolTipId={`${field}-error-tooltip}`} toolTip={errorTooltip}>
+        <Row className="align-items-center">
+          <Col>
+            <StyledDiv className={!!touch && !!error ? 'is-invalid' : ''}>
+              {value?.summary}
+            </StyledDiv>
+            <Input field={field + '.id'} className="d-none"></Input>
```

Comment:
wait, isn't this the numeric id from the database? Is there value to including this here?

Reply:
This was necessary to get the tests running, Is quite hacky, but I couldn't think of a different way of triggering the yum validation when doind the jest test